### ISSUE TITLE
"Download XML" link leads to www.osm instead of api.osm

### ISSUE THREAD

**Issue by Zverik** [openstreetmap/openstreetmap-website#1244]
See for example [this node history](https://www.openstreetmap.org/node/4233999190/history). The "Download XML" link at the bottom leads to https://www.openstreetmap.org/api/0.6/node/4233999190/history while it should go to api.openstreetmap.org.

**Comment by tomhughes** [comment]
Says who? The two are equivalent and to date we have not made any particular effort to separate them. When we want to do that we will fix the links.

**Comment by Zverik** [comment]
I've found a note somewhere in the wiki that while `www.` and `api.` point to the same server now, they may be separated in the future, so it is recommended to use the proper subdomain. I though that it would be nice for osm.org to set an example.

**Comment by tomhughes** [comment]
That is correct, but it's non-trivial to fix, so while third parties should certainly do so there is no immediate urgency for us to do so.

I'll reopen this as a placeholder though, and retitle it to make it more general.
